### PR TITLE
Closes #1 - Create index.html

### LINKED ISSUE #1: Create index.html

### PR DESCRIPTION
**Pull request by annakynutsya**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+pls approove
```
Comment (annakynutsya) [inline index.html:1]: aaaaa

Comment (kukurudz-nadia) [inline index.html:1]: new comment

**Review by kukurudz-nadia — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (BarryLev) [inline index.html:1]: eeeeee

**Review by BarryLev — CHANGES_REQUESTED** [review]

**Review by BlackoutForever — CHANGES_REQUESTED** [review]
add more code

File: index.html (hunk 1 above)
Comment (Userlastname) [inline index.html:1]: а

**Review by kukurudz-nadia — APPROVED** [review]

**Review by Userlastname — APPROVED** [review]

**Review by BlackoutForever — APPROVED** [review]

**Review by heroisaprinciple — APPROVED** [review]
cool

**Review by BarryLev — APPROVED** [review]

**Pull request merged**